utils/_common.py

```diff
@@ -54,6 +52,22 @@ def get_logger(initial_level=logging.INFO):
     return logger
 
 
+def set_logging_level(verbose=False, quiet=False):
+    """Sets logging level of logger and all its handlers"""
+
+    default_level = logging.INFO
+    logging_level = default_level + 10 * (quiet - verbose)
```

Comment:
This is a clever and succinct solution, but I think this is not great from a user perspective because:

1. The verbose and quiet flags have an implicit dependency on each other. And when both are applied, surprising behavior occurs (based on the help messages you wrote)
2. There are no comments to describe how this works; e.g. it is implicit that the value `10` will change it to different logging levels. Also, is this behavior a guarantee from the Python API, or just an implementation detail?

I think the better solution is to let the user decide which logging level they want. Either a numerical value or a string to specify the logging level would make things more clear (maybe even accept both)?

Reply:
The values are in Python documents so it should be guaranteed, but I think you are right, it will be better for users to specify the logging level. I think I can just make verbose and quiet alias of some logging levels.